cmd/machine-healthcheck/main.go

```diff
@@ -24,8 +26,24 @@ func printVersion() {
 }
 
 func main() {
-	watchNamespace := flag.String("namespace", "", "Namespace that the controller watches to reconcile machine-api objects. If unspecified, the controller watches for machine-api objects across all namespaces.")
-	metricsAddress := flag.String("metrics-bind-address", metrics.DefaultHealthCheckMetricsAddress, "Address for hosting metrics")
+	watchNamespace := flag.String(
+		"namespace",
+		"",
+		"Namespace that the controller watches to reconcile machine-api objects. If unspecified, the controller watches for machine-api objects across all namespaces.",
+	)
+
+	metricsAddress := flag.String(
+		"metrics-bind-address",
+		metrics.DefaultHealthCheckMetricsAddress,
+		"Address for hosting metrics",
+	)
+
+	healthAddr := flag.String(
+		"health-addr",
+		":9442",
```

Comment:
This is a raw number here but it's a constant here https://github.com/openshift/machine-api-operator/pull/602/commits/6c8fa986357762bd211425ea54200bcbc558c429#diff-fa45321336db7ad1cedc28bf643a4f97R34
can't we have a constant that we use everywhere?

Reply:
I can do this only for that PR without introducing circular dependency and having forcefully revendor MAO from a local branch for each of 5 pending provider implementation. I'm going to change it only here for now, if we still want to merge it all at once.

Reply:
Also, in order to do this change I'll have to remove the rest of `glog` usages, or it breaks flag initialization.

Reply:
I can't do this right now. Let's prioritize `glog` usage removal.

Reply:
I'm not sure how this relates to any revendor. The code I pointed live all in this repo.
Any way, not a blocker to me.

Reply:
+1, i think making this a constant would be a good improvement for a followup